src/haxelib/client/Main.hx

```diff
@@ -1344,6 +1345,16 @@ class Main {
 		}
 	}
 
+	function libpath( ) {
+		var rep = getRepository();
+		while( argcur < args.length ) {
+			var a = args[argcur++].split(":");
+			var results = new List();
+			checkRec(rep, a[0],a[1],results);
+			if( !results.isEmpty() ) Sys.println(results.first().dir);
```

Comment:
Why only `results.first()`?

Reply:
Well, I suppose the question is again what exactly you want. Assuming you really want just the one path (and not the dependencies), this is fine. But then looping the args is nonsense. Instead you should make sure there's exactly one argument left and process just that.

Reply:
`results.first()` will give you the library's directory, which `haxelib libpath` should return. Any other paths will be dependency paths.

The looping is replicating the behaviour from `haxelib path openfl lime` - if you run with `haxelib libpath openfl lime` it will return the paths for both libraries.

If you only run it with `haxelib libpath openfl` you will only get one path.

Since one `path` method works that way, I didn't want to make it confusing by changing the behaviour of the `libpath` method. The end-user doesn't have to supply multiple arguments.